changes to the grammar.md

```diff
@@ -0,0 +1,27 @@
+# Changes to the official grammar of the I language by the noname team
+## * Expression breakdown:
+```
+Expression ::= ( 'not' Expression | Comparison [ ( 'and' | 'or' | 'xor' ) Expression ] )
+Comparison ::= Sum [ ( '<' | '<=' | '>' | '>=' | '=' | '/=' ) Comparison ]
+Sum ::= Product [ ( '+' | '-' ) Sum ]
+Product ::= Prioritized [ ( '*' | '/' | '%' ) Product ]
+Prioritized ::= ( '(' Expression ')' | Primary )
+Primary ::= ( '-' Primary | '+' Primary | IntegerLiteral | RealLiteral | BooleanLiteral | ModifiablePrimary | RoutineCall )
+```
+## + Return statement:
+```
+ReturnStatement ::= ‘return’ Expression
+```
+## + Specification of type convertion from real to boolean:
```

Comment:
Nah, the spec explicitly says that converting from real to boolean is illegal, we cannot change that

Reply:
Then `ensureType` needs to be fixed(

Reply:
Do it in another PR